test/campaigns.js

```diff
@@ -37,21 +36,23 @@ describe('campaigns', function() {
     const hubspotDemo = new Hubspot({ apiKey: 'demo' })
 
     describe('successfully', function() {
-      let campaignId
+      let campaignId = 345
+      const campaignGetEndpoint = {
+        path: `/email/public/v1/campaigns/${campaignId}`,
+        response: { id: campaignId },
+      }
+      fakeHubspotApi.setupServer({
+        demo: true,
```

Comment:
I'm uncertain why the `hapikey` needs to be set for this endpoint but not for the other endpoint, like `campaignsGetEndpoint`. Can you shed some light on the use of `demo: true`?

Reply:
Sure. I couldn't work out how to get email campaigns on my testing server so I decided to leave this on the demo account.